acct.bionimbus.org/portal/gitops.json

```diff
@@ -79,253 +93,329 @@
         {
           "icon": "dictionary",
           "link": "/DD",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "Dictionary"
         },
         {
           "icon": "exploration",
-          "link": "/files",
-          "color": "#a2a2a2",
+          "link": "/explorer",
+          "color": "#1a3057",
           "name": "Exploration"
         },
         {
           "icon": "query",
           "link": "/query",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "query"
         },
         {
           "icon": "workspace",
-          "link": "/workspace",
-          "color": "#a2a2a2",
+          "link": "#hostname#workspace/",
+          "color": "#1a3057",
           "name": "Workspace"
         },
         {
           "icon": "profile",
           "link": "/identity",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "Profile"
         }
       ]
     },
-    "topBar": {
-      "items": [
-        {
-          "icon": "upload",
-          "link": "/submission",
-          "name": "Browse Data"
-        },
-        {
-          "link": "https://gen3.org/resources/user/",
-          "name": "Gen3 Documentation"
-        }
-      ]
-    },
     "login": {
-      "title": "ACCOuNT Data Commons",
+      "title": "ACCOuNT Data Commons (QA)",
       "subTitle": "Explore, Analyze, and Share Data",
       "text": "This website provides a centralized, cloud-based discovery portal for African American pharmacogenomics data and aims to accelerate discovery of novel genetic variants in African Americans related to clinically actionable cardiovascular phenotypes.",
-      "contact": "If you have any questions about access or the registration process, please contact support@datacommons.io"
+      "contact": "If you have any questions about access or the registration process, please contact ",
+      "email": "support@datacommons.io",
+      "image": "/../../../../custom/sponsors/gitops-sponsors/gene_br2"
     },
+    "categorical9Colors": [
+      "#1a3057",
+      "#C62C2E",
+      "#FEBA00",
+      "#FF8800",
+      "#2E59A2",
+      "#FFD900",
+      "#E76662",
+      "#95B5E5",
+      "#F3B44F"
+    ],
+    "categorical2Colors": [
+      "#1a3057",
+      "#C62C2E"
+    ],
     "footerLogos": [
       {
-        "src": "/custom/sponsors/gitops-sponsors/nimhd.jpg",
+        "src": "/src/img/sponsors/nimhd.png",
```

Comment:
No.

Reply:
Oh, I see the problem here 😓 , let me prepare a fix so the automation will not replace that value in the next run.